contracts/src/modules/HarvesterPlugin.sol

```diff
@@ -95,7 +186,8 @@ contract DummyModule is BaseModule {
 
     SafeTransaction memory transaction = SafeTransaction({
       actions: transactions,
-      nonce: 0,
+      // NOTE: find uniqueness via `block.number`
+      nonce: uint256(keccak256(abi.encode(address(_safe), block.number))),
```

Comment:
does this work? shouldnt the nonce be sequential?

Reply:
what i understood is that is sufficient is to be unique not necessarily sequential

Reply:
ah ok so this is not the actual tx nonce.

assumption here is then that this `SafeTransaction` is the only one in this block!